pkg/installer/custominstallconfig.go

```diff
@@ -49,9 +49,6 @@ const (
 var (
 	targetAssets = []asset.WritableAsset{
 		&cluster.Metadata{},
-		&machine.MasterIgnitionCustomizations{},
-		&machine.WorkerIgnitionCustomizations{},
-		&tfvars.TerraformVariables{},
```

Comment:
I think we should be able to use the [IgnitionConfigs target](https://github.com/openshift/installer/blob/main/pkg/asset/targets/targets.go#L53-L62) directly instead of duplicating its contents here.

See https://issues.redhat.com/browse/ARO-13145 for analysis.

Reply:
Did this, and it appears to be all good